ui/src/app/streams/stream-definitions/stream-definitions.component.html

```diff
@@ -98,10 +98,16 @@
                   class="btn btn-default" style="margin-left: 0;" title="Undeploy">
             <span class="glyphicon glyphicon-stop"></span>
           </button>
-          <button type="button" [appRoles]="['ROLE_CREATE']" [disabled]="(item.status==='deployed' || item.status==='deploying')" (click)="deploy(item)"
+
+          <button *ngIf="!(featureInfo | async)?.skipperEnabled" type="button" [appRoles]="['ROLE_CREATE']" [disabled]="(item.status==='deployed' || item.status==='deploying')" (click)="deploy(item)"
                   class="btn btn-default" style="margin-left: 0;" title="Deploy">
             <span class="glyphicon glyphicon-play"></span>
           </button>
+          <button *ngIf="(featureInfo | async)?.skipperEnabled" type="button" [appRoles]="['ROLE_CREATE']" class="btn btn-default" style="margin-left: 0;"
```

Comment:
According to the story this should be disabled. but after chatting with @sabbyanandan the use of the message on the active button was agreed solution.   No change required, just wanted to make a note of it.